test/Microsoft.AspNet.Mvc.Core.Test/Rendering/DefaultTemplatesUtilities.cs

```diff
@@ -141,7 +175,12 @@ public static HtmlHelper<TModel> GetHtmlHelper<TModel>(TModel model, ICompositeV
             ICompositeViewEngine viewEngine,
             IModelMetadataProvider provider)
         {
-            return GetHtmlHelper(model, urlHelper, viewEngine, provider, innerHelperWrapper: null);
+            return GetHtmlHelper(
+                model,
+                urlHelper,
+                viewEngine,
+                provider,
+                innerHelperWrapper: null);
```

Comment:
why did this change and the one above happen?  one line is fine when everything fits.